### DISCUSSION TITLE
Stuck on step#2 as well

### DISCUSSION THREAD

**Discussion by ghost** [skills/.github#603] [Communicate using Markdown]
Run skills/action-check-file@v1
Run echo "Check that all required env variables are set"
Check that all required env variables are set
Check that index.md includes (?<!#)#{1,6}([ \t](.*))?$
Missing (?<!#)#{1,6}([ \t](.*))?$ in index.md
----------------
Error: Process completed with exit code 204.

How do I modify the index.md file? do copy it out of GIT? does the Pull request send it to my PC? 

In this pull request, go to the Files changed tab. We made an empty file index.md for you.
Select Edit file from the three dotted ... menu in the upper right corner of the file view on index.md.
On the Edit file tab, add a #, followed by a space, before any content you like to make it an H1 Header. You can add more headers, using one to six # characters followed by a space.

the three button drop down does not have edit choice.

![image](https://github.com/skills/.github/assets/173011312/cff904af-498c-4c63-a469-ba4c513423a5)

**Comment by mjbear** [comment]
Might be this https://github.com/orgs/skills/discussions/330#discussioncomment-6776297

**Update:** The key is the Github Actions workflows which [hint at which branch they hook in on](https://github.com/skills/communicate-using-markdown/blob/main/.github/workflows/1-add-headers.yml#L12). Live and learn!
